maestro-client/src/main/java/maestro/debuglog/IOSDriverLogger.kt

```diff
@@ -0,0 +1,13 @@
+package maestro.debuglog
+
+import ios.logger.Logger
+import maestro.drivers.IOSDriver
+
+class IOSDriverLogger : Logger {
+
+    private val logger by lazy { DebugLogStore.loggerFor(IOSDriver::class.java) }
```

Comment:
Should this be `IOSDriverLogger`?
```suggestion
    private val logger by lazy { DebugLogStore.loggerFor(IOSDriverLogger::class.java) }
```

Reply:
Nice catch! Done ✅

Reply:
Nice catch! Done